config/config_schema.json

```diff
@@ -101,7 +111,16 @@
                 "type": "string",
                 "enum": ["isomir", "mirna", "pri-mir"]
             },
-            "default": ["isomir", "mirna", "pri-mir"]
+            "default": ["isomir", "mirna", "pri-mir"],
+            "description": " miRNA speices to be quantified."
+        },
+        "lib_dict":{
+            "type": "object",
+            "additionalProperties":{
+                "type": "array",
+            },
+            "default": {},
+            "description": "Dictionary with a list of library names, as in the samples table, to generate pileups for as value and a condition name as keys."
```

Comment:
I find this sentence very hard to parse. Maybe:

Dictionary of arbitrary condition names (keys) and library names to aggregate alignment pileups for (values; MUST correspond to names in samples table).